locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
Oh, and I just remembered that I prefer the `<term></term>` format because it's clearer to translations that we're dealing with an empty string. It's also easier for translations to turn `<term></term>` into `<term>blah</term>` than it is to turn `<term/>` into `<term>blah</term>` (the latter requires more editing and some understanding of HTML/XML tags).

It might be easier to fix this up for the entire repo, though, with a find-and-replace (basically "/>" to "></term>", although that would certainly mess up `<style-options/>` so some fancier RegEx would be needed). I can just merge this as is.